doc: Remove info about migrating active sessions, match the branding (#443)
TheThingsNetwork/docs: doc/content/the-things-stack/_index.md

## How can I access V3?

> The Things Network, which is a (free) community edition of The Things Stack V3, can be accessed via Console. You can access it by <a href="https://console.cloud.thethings.network/" target="blank_">selecting a cluster</a> which is closest to you geographically.

Reviewer: ```suggestion
The Things Stack Community Edition, which is a (free) community edition of The Things Stack, can be accessed via the Console. You can access it by <a href="https://console.cloud.thethings.network/" target="blank_">selecting a cluster</a> which is closest to you geographically.
```

> The (free) community edition of The Things Stack V3 can be accessed via the <a href="https://console.cloud.thethings.network/" target="_blank">**Console**</a>. Make sure to <a href="https://console.cloud.thethings.network/" target="blank_">select the cluster</a> which is closest to you geographically.

Reviewer 2: Probably needs new naming as well; `The Things Stack Community Edition by The Things Network` ?

With or without Edition?

> ![The Things Network Console](TTN-V3-console.png "The Things Network Console")

Reviewer: ```suggestion
![The Things Stack Community Console](TTN-V3-console.png "The Things Stack Community Console")
```

> ![The Things Network Console](TTN-V3-console.png "The Things Network Console")

Reviewer 2: @benolayinka @laurensslats what is the final name here?

Reviewer: @nejraselimovic The name changed from `The Things Network` to `The Things Stack Community`

Reviewer: ```suggestion
![The Things Stack Community](TTN-V3-console.png "The Things Stack Community Console")
```

## Why should I migrate from V2 to V3?

> Here, we assume you have experience using The Things Network running The Things Network Stack V2. If you have not been using The Things Network yet, you can visit the [The Things Stack Quick Start]({{< ref "quick-start" >}}).

> The Things Stack V3 is more scalable, more secure and supports all the latest LoRaWAN developments like the latest LoRaWAN versions 1.1 and 1.0.4. Near the end of 2021, The Things Network V2 clusters will be shut down. 

Reviewer 2: So we're calling it The Things Stack V3, The Things Network V2

And we also have The Things Stack Community Edition that is The Things Network.

@laurensslats @benolayinka even I am getting confused now.

> The Things Stack V3 is more scalable, more secure and supports all the latest LoRaWAN developments like the latest LoRaWAN versions 1.1 and 1.0.4. Near the end of 2021, The Things Network V2 clusters will be shut down. 

Reviewer: ```suggestion
The Things Stack, compared to The Things Network V2, is more scalable, more secure and supports all the latest LoRaWAN developments like the latest LoRaWAN versions 1.1 and 1.0.4. Near the end of 2021, The Things Network V2 clusters will be shut down. 
```
